### DISCUSSION TITLE
16:9 Widescreen

### DISCUSSION THREAD

**Discussion by charliefiddlesticks** [ScratchAddons/ScratchAddons#6628] [Ideas]
I had this cool idea that lets you make the Scratch stage 16:9 for the record feature. This would work wonders for people like me that want to make a wider Scratch animation video. Of course this wouldn't be visible by other Scratch users and would just be the default widescreen

**Comment by Samq64** [comment]
This wouldn't work well since projects are almost always designed for 4:3 and if you make a widescreen one it wouldn't look right for other users.[TurboWarp](https://turbowarp.org) has this though.
